lib/core/src/display/shape/text/text_field.rs

```diff
@@ -94,6 +94,21 @@ shared! { TextField
             self.display_object.set_position(position);
         }
 
+        /// Get position of this TextField.
+        pub fn position(&self) -> Vector3<f32> {
```

Comment:
Why is this `Vector3`, while many orther places use `Vector2` and both are being referred to as "position"?
Do we have like two kinds of components / contexts and only one cares about Z coordinate?

Reply:
@mwu-tow can you point out the other places were position is Vector2<f32>? TextField is a DisplayObject, so it has a 3D position.

Reply:
The `UiComponent` uses `Vector2` for position. Is this property only of "display objects" that they have Z coordinate? What is relation between "ui component" and "display component"? Why it should never care about Z ordering?
(some of these questions might be worth answering in the doc comment for `UiComponent`)